### PR TITLE
feature: element summary

### PR DESCRIPTION
**Pull request by Midnighter**
* Make the context test for the model summary closer to what was reported in #900 
* Add the ability to display atomic element uptake/secretion to the model summary. See https://gist.github.com/Midnighter/84590cbc5d46209897abcf6ec2ba812d for a preview

### CODE REVIEW

File: src/cobra/summary/model_summary.py [hunk 1]
```diff
@@ -396,7 +419,11 @@ def to_string(
         )
 
     def to_html(
-        self, names: bool = False, threshold: float = 1e-6, float_format: str = ".4G"
+        self,
+        names: bool = False,
+        element: str = "C",
+        threshold: float = 1e-6,
```
Comment (cdiener) [inline src/cobra/summary/model_summary.py]: I wonder if we should accept a `None` argument for threshold that initializes it to the model tolerance. This threshold would make little sense if the model tolerance is 1e-3 for instance. Can be left for a future PR though.

**Review by cdiener — COMMENTED** [review]
Looks good. I love having usage previews in PRs. Kudos!

**Review by cdiener — APPROVED** [review]

File: src/cobra/summary/model_summary.py (hunk 1 above)
Comment (Midnighter) [reply src/cobra/summary/model_summary.py]: I would have loved to do that but I wanted to structure the summary in a way that at the time of display, it does not access the model any longer. However, I could copy the model tolerance on initialization and store that value for later.

Comment (Midnighter) [reply src/cobra/summary/model_summary.py]: Okay, I have implemented something like what you asked for.

**Pull request merged**